modules/microshift-install-rpm-before.adoc

```diff
@@ -16,6 +16,6 @@ Preparation of the host machine is recommended prior to installing {microshift-s
 To configure a volume group (VG) that allows LVMS to create the LVs for your workload's PVs, lower the *Desired Size* of your root volume during the installation of {op-system}. Lowering the size of your root volume allows unallocated space on the disk for additional LVs created by LVMS at runtime.
 
 [id="microshift-running-containers-fips-mode_{context}"]
-== Running {microshift-short} in FIPS-compliant mode
+== Running {microshift-short} in FIPS mode
```

Comment:
Consider using a different title. The style guide discourages gerunds in concepts https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=Avoid%20using%20gerunds%20in%20concept%20titles.%20%22About%20%3Cconcept%3E%22%20is%20a%20common%20concept%20module%20title